src/shared/sleep-config.c

```diff
@@ -266,13 +270,13 @@ static int read_resume_files(dev_t *ret_resume, uint64_t *ret_resume_offset) {
 
         r = read_one_line_file("/sys/power/resume_offset", &resume_offset_str);
         if (r == -ENOENT)
-                log_debug("Kernel does not support resume_offset; swap file offset detection will be skipped.");
+                log_debug_errno(r, "Kernel does not support resume_offset; swap file offset detection will be skipped.");
```

Comment:
We usually want to pass along error codes we receive. Note that log_debug_errno() synthesizes structured journal log records with ERRNO= set to the first parameter. i.e. we pass that simply because we have the rule to pass along all error codes we receieve. We do exclude it from the actual log message though, i.e. there's no `%m` in it , since it's not intresting since it is a single specific error code.

Hope that makes sense?

Reply:
Ugh, my bad, I didn't notice you left out the %m. I didn't know about the ERRNO stuff, that's good to know.